tests/test_etter_filter.py

```
@@ -0,0 +1,220 @@
+        assert f.buffer_distance_m is None
+        assert f.confidence == 1.0
+
+    def test_frozen(self):
+        gq = SimpleNamespace(
+            spatial_relation="in",
+            reference_location="X",
+            buffer_config=None,
+            overall_confidence=1.0,
+        )
+        f = ef.EtterFilter.from_geoquery("q", gq)
+        with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)